Gateway/Request/CreditCardBuilder.php

```diff
@@ -22,8 +23,13 @@ public function build(array $buildSubject)
         $payment = SubjectReader::readPayment($buildSubject);
         $method  = $payment->getPayment();
 
-        return [
-            self::CARD => $method->getAdditionalInformation(CreditCardDataObserver::TOKEN)
-        ];
+        if ($method->getAdditionalInformation(CreditCardDataObserver::CUSTOMER)) {
```

Comment:
(note) on both 'using an existing card' or 'save a new card', `OmiseCustomer.id` will always be presented inn the payment method's additional information.